src/guide/creating-new-projects.md

```diff
@@ -0,0 +1,95 @@
+
+Most Vue components can be written using either the Options API or the Composition API, which brings us to the question of which to use. There are two main approaches commonly seen in real applications.
+
+The first approach is to forgo the Options API entirely and just use the Composition API.
+
+The second approach is to use the Options API by default and to only use the Composition API when there's no other option, e.g. because a third-party library requires it.
+
+If you aren't sure which approach to take, we suggest that you try the Composition API. Using `<script setup>` is also highly recommended, as it cuts away most of the boilerplate.
+
+You might prefer to use the Options API if you don't have a build process and can't use single-file components. You can still use the Composition API in that scenario, but some of the elegance is lost without `<script setup>`.
+
+If your team has used the Options API on previous projects, then familiarity may be an important factor when choosing which way to go on a new project. Code written using the Composition API does not use the `this` keyword to access component properties. Instead, it makes greater use of plain functions and closures. It can feel a little alien at first, especially if you're used to the Options API.
+
+We occasionally receive questions about whether the Options API is deprecated. It isn't. It remains very popular within the community and will continue to be supported so long as that remains the case. As a mature API, it's unlikely to need any major new features, but we remain open to suggestions for further enhancements. We do encourage you to migrate away from mixins though, even within components that use the Options API. Replacing the `mixins` option with a call to a composable is usually a relatively small change that shouldn't impact the rest of the component.
+
+## Should I use `<script setup>`?
+
+If you're using the Composition API with single-file components then `<script setup>` is the recommended approach.
+
+## Why does the documentation start with the Options API?
+
+We're in the process of rewriting the documentation to give both APIs equal visibility.
+
+Prior to the introduction of `<script setup>`, the Options API was usually an easier way to start learning Vue. With the addition of `<script setup>` in Vue 3.2, it is now possible to introduce the Composition API at a much earlier stage in the documentation.
+
+The current guides introduce Vue using a global/CDN approach. Avoiding tooling allows us to jump straight in with writing code, without worrying about whether the reader has prior experience of tools like `npm` or `webpack`. That decision made sense when the documentation was first written, but it increasingly makes the documentation feel disconnected from how real Vue applications are built. The new documentation should also help to tackle this problem.
```

Comment:
Slightly unrelated side-note: I 100% agree with this feel of disconnectedness, it was by far the most difficult part of teaching people Vue last summer.